libexec/pyenv

```diff
@@ -7,7 +7,8 @@ if [ "$1" = "--debug" ]; then
 fi
 
 if [ -n "$PYENV_DEBUG" ]; then
-  export PS4='+ [${BASH_SOURCE##*/}:${LINENO}] '
+  # https://wiki-dev.bash-hackers.org/scripting/debuggingtips#making_xtrace_more_useful
```

Comment:
I added it so that ppl know where this strange incantation comes from and thus what to expect of it (=it's a solid, time-proven general-purpose solution rather than someone's bad trip).

This is a "specter" ("призрак") in N.N.Nepeivoda's terms: additional knowledge which is critical for solving the problem but which is impractical or impossible to express in the code proper. So Nepeivoda argues that those should be expressed in comments instead. Check out his lecture "призраки и подпорки" (specters and supports) to learn more.